### ISSUE TITLE
No folders selected

### ISSUE THREAD

**Issue by nmze** [mailcow/mailcow-dockerized#4448]
Does not work for me specifically, but in general, in principle, works for. Simply install by instructions and transfer to a local network isolated from the Internet and everything, nothing works, writes: "Not a single folder is selected."

**Comment by MAGICCC** [comment]
Logs? Where? What? Who?

Please dont delete the issue template.

**Comment by nmze** [comment]
I described the problem: I put it as written in the instructions and everything works. I disable this VM access to the Internet and if you do not reboot it, it works until the first reboot, if you simply turn on the mail "No folders selected" without Internet access
I only decided today to try "mailcow" and immediately after installation came across this problem.

**Comment by btzdnl** [comment]
I believe English is not your first language. It is rather hard to follow your description of where you see what problem exactly.

Here’s the facts I gather:

* You set up Mailcow following the official install instructions
* You then took the VM where you installed it and put it in a different network without internet access

Then you see some error. But what were you trying to do? Where, exactly, do you see the error? In your mail client software? Or maybe in SOGo? If possible, please provide a screenshot of the error.

**Comment by nmze** [comment]
How do I reproduce the problem? Very simple!
We put everything on instructions, everything works! Without a reboot, we change the network to an isolated network from the Internet - everything continues to work, but if you reboot - the mail does not work, writes "No folders selected"
Or turn off, transfer to a network isolated from the Internet - mail does not work, writes "No folders selected"
At first I thought that I was doing something wrong, four times I did everything again, the result is the same, without the Internet: "No folders selected"
Then the question? How do I run an Internet-isolated LAN?

**Comment by ghost** [comment]
Mailcow is a complete email suite that needs an internet access to work. You can't run Mailcow without internet.

**Comment by milkmaker** [comment]
This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.
